misk/src/test/kotlin/misk/config/ConfigTest.kt

```diff
@@ -67,23 +67,30 @@ class ConfigTest {
 
     @Test
     fun friendlyErrorMessagesWhenFilesNotFound() {
-        assertThrows(IllegalStateException::class.java, {
+        val exception = assertThrows(IllegalStateException::class.java, {
```

Comment:
why the change?

Reply:
This method doesn't behave as expected. If you dig into the assertThrows impl, if the exception is the same as the argument passed, the string provided is ignored.